homeassistant/components/binary_sensor/doorbird.py

```diff
@@ -0,0 +1,83 @@
+"""Support for reading binary states from a DoorBird video doorbell."""
+from datetime import timedelta
+import logging
+import voluptuous as vol
+
+from homeassistant.components.binary_sensor import BinarySensorDevice,\
+    PLATFORM_SCHEMA
+from homeassistant.components.doorbird import DOMAIN
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_MONITORED_CONDITIONS, STATE_UNKNOWN
+from homeassistant.util import Throttle
+
+DEPENDENCIES = ['doorbird']
+
+_LOGGER = logging.getLogger(__name__)
+_MIN_UPDATE_INTERVAL = timedelta(milliseconds=250)
+
+SENSOR_TYPES = {
+    "doorbell": {
+        "name": "Doorbell Ringing",
+        "icon": {
+            True: "bell-ring",
+            False: "bell",
+            STATE_UNKNOWN: "bell-outline"
+        }
+    }
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_MONITORED_CONDITIONS, default=[]):
+        vol.All(cv.ensure_list([vol.In(SENSOR_TYPES)]))
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the DoorBird binary sensor component."""
+    device = hass.data.get(DOMAIN)
+
+    sensors = []
+    for sensor_type in config.get(CONF_MONITORED_CONDITIONS):
+        if sensor_type == "doorbell":
```

Comment:
`doorbell` is the only condition with is allowed by the configuration validation at the moment.